src/graph_gui.cpp

```diff
@@ -376,7 +376,9 @@ struct BaseGraphWindow : Window {
 		/* Draw the background of the graph itself. */
 		GfxFillRect(r.left, r.top, r.right, r.bottom, GRAPH_BASE_COLOUR);
 
-		/* Draw the vertical grid lines. */
+		/* Draw the grid lines. */
+		int gridline_width = WidgetDimensions::scaled.bevel.top;
```

Comment:
Should we pick `bevel` or `fullbevel` here? `bevel` is dependent on the "Scale bevels" option, whereas `fullbevel` is always scaled by interface scale.